### ISSUE TITLE
Clients specifying @FormParam (for x-www-form-urlencoded) does not work

### ISSUE THREAD

**Issue by danpfe** [kumuluz/kumuluzee-rest-client#20]
Think I discovered a bug in this module.

When specifying a client interface as following:

```
    @POST
    @Produces(MediaType.APPLICATION_JSON)
    @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
    OpAuthResponse login(@FormParam("grant_type") String grantType,
                         @FormParam("client_id") String clientId,
                         @FormParam("client_secret") String clientSecret,
                         @FormParam("username") String username,
                         @FormParam("password") String password);
```

the call never succeeds as the module forcefully changes the MediaType to multipart/form-data. Looking at the code of this module neither `ParamInfo` nor `RestClientInvoker` seem to treat FormParams any differently than FormDataParams.

Is this intentional?

**Comment by Jamsek-m** [comment]
Hi,

Indeed, @FormParam was handled as equivalent to @FormDataParam (jersey multipart), which was incorrect - #21 should fix this.

**Comment by danpfe** [comment]
Great, thanks for the quick fix. Hoping for a quick publish :)

**Comment by urbim** [comment]
@danpfe This is released in v2.0.0.
